server/routers/exercise/queries/get-user-streak.ts

```
@@ -0,0 +1,83 @@
+        where: and(
+            eq(submission.userId, userId),
+            eq(submission.isStreak, true)
+        ),
+        orderBy: desc(submission.createdAt),
+        columns: {
+            createdAt: true,
+        },
+    });
+
+    const streak = calculateStreak(submissions);
+const streak = calculateStreak(submissions);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'streak'. Consider to delete it or rename it.

'streak' is defined here:


(lint/suspicious/noRedeclare)